sklearn/neural_network/_multilayer_perceptron.py

```diff
@@ -793,11 +793,10 @@ class MLPClassifier(ClassifierMixin, BaseMultilayerPerceptron):
         Whether to shuffle samples in each iteration. Only used when
         solver='sgd' or 'adam'.
 
-    random_state : int, RandomState instance or None, default=None
-        If int, random_state is the seed used by the random number generator;
-        If RandomState instance, random_state is the random number generator;
-        If None, the random number generator is the RandomState instance used
-        by `np.random`.
+    random_state : int, RandomState, default=None
+        Determines random number generation for MLPC Classifier. Pass an
```

Comment:
This needs a description on how this estimator uses randomness. For example MLP uses the the generator to initialize weights and to split data when early stopping is active.